common/invoke.go

```diff
@@ -0,0 +1,31 @@
+package common
+
+import (
+	"github.com/nspcc-dev/neo-go/pkg/interop/crypto"
+	"github.com/nspcc-dev/neo-go/pkg/interop/runtime"
+)
+
+func InvokeID(args []interface{}, prefix []byte) []byte {
```

Comment:
`InvokeID` used to identify txs and collect votes, so it better to move to `vote.go`.